.github/workflows/ci.yml

```diff
@@ -23,6 +23,7 @@ jobs:
           source .env/bin/activate
           python -m pip install -U pip
           make install-dev
+          sudo apt-get install -y ffmpeg
```

Comment:
note sure if this is the best way to do this

Reply:
Is it not included when pip installing?